xItemSrc/xItemLib_v112.lua

```diff
@@ -154,14 +154,18 @@ local function smuggleDetection()
 	end
 	
 	for i=1, #group
-		if group[i].kartstuff[k_position] <= 2
-		and (group[i].kartstuff[k_itemtype] == 1
-		or group[i].kartstuff[k_itemtype] == 2
-		or group[i].kartstuff[k_itemtype] == 3
-		or group[i].kartstuff[k_itemtype] == 11
-		or group[i].kartstuff[k_invincibilitytimer] > 0
-		or group[i].kartstuff[k_growshrinktimer] > 0
-		or (HugeQuest and group[i].hugequest.huge > 0))
+		if 
+			group[i].kartstuff[k_position] <= 2
```

Comment:
would it be smart to use xitem ids here instead of hardcoded numbers?

Reply:
You probably wanna have further cleanup in a different PR, I'd rather have this merged now for the cvar.

Reply:
sounds good, merging...